ambrs/ppe.py

```diff
@@ -29,6 +29,8 @@ class EnsembleSpecification:
     flux: RVFrozenDistribution
     relative_humidity: RVFrozenDistribution
```

Comment:
PartMC prescribes the initial RH, which can then vary over time as temperature varies.